app/models/content_item.rb

```diff
@@ -43,9 +45,8 @@ def add_taxonomies_by_id(taxons)
 private
 
   def linked_content(link_type)
-    search = Search.new
-    search.filter_by(link_type: link_type, source_ids: content_id)
-    search.execute
-    search.content_items
+    links = Link.where(link_type: link_type, source_content_id: content_id)
+
+    ContentItem.where(content_id: links.select(:target_content_id))
```

Comment:
What's the reason for this change? I think it's nice that `Search` encapsulates the knowledge of how `links` relate to `content_items`. If we do this, then that knowledge is in two places. Maybe that's not a problem? What do you think?

If we do keep this, `#pluck` might be more efficient so we skip constructing AR objects for each link.